html/js/product-search.js

```diff
@@ -143,45 +145,50 @@ function display_products(target, product_groups ) {
 		
 		$.each( product_group, function(key, product) {
 		
-			var product_html = "<li>";
+			var product_html = "";
 			
-			product_html += '<a href="' + product.url + '"><div>';
+			var color = '#eee';
+			if (product.match_status == "yes") {
+				color = '#cfc';
+			}
+			else if (product.match_status == "no") {
+				color = '#fcc';
+			}
+						
+			product_html += '<li><a href="' + product.url + '" style="background-color:' + color + ';border-radius:12px;padding:1rem;display:block;">';
+			product_html += '<div style="height:100px;line-height:100x;text-align:center;">';
 			
 			if (product.image_front_thumb_url) {
-				product_html += '<img src="' + product.image_front_thumb_url + '">';
+				product_html += '<img src="' + product.image_front_thumb_url + '" style="display:block:margin-left:auto;margin-right:auto;height:auto;display:inline-block;vertical-align:middle;">';
 			}
 			
 			product_html += "</div>";
 			
-			product_html += "<span>" + product.product_name + "</span>";
-			
-			product_html += '</a>';
-			
+			product_html += '<div style="height:80px;overflow:hidden;">' + product.product_name + "</div>";
+									
 			$.each(product.match_icons.mandatory.concat(product.match_icons.very_important, product.match_icons.important), function (key, icon_url) {
 				
-				product_html += '<img src="' + icon_url + '" class="match_icons">';
+				product_html += '<img src="' + icon_url + '" style="height:24px;margin-right:0.2rem;margin-top:0.2rem;">';
```

Comment:
will the style= stuff go into classes at some point?

Reply:
Yes, once the styles are finalized we will definitely move them. To tinker with the styles I find it much easier to have them close to the HTML and somewhere where the effects are immediate without having to rebuild the scss stuff etc.

Reply:
Fixed.